app/actions/user.ts

```diff
@@ -104,24 +104,33 @@ export async function getFollowees(
     }
 }
 
-export async function followUser(
-    userId: string,
-    followType: FollowType,
-): Promise<ActionResponse<void>> {
+export async function followUser(formData: FormData): Promise<string> {
+    const userId = formData.get('userId');
+    const isFollowingString = formData.get('isFollowing');
+
+    if (
+        typeof userId !== 'string' ||
+        (isFollowingString !== 'true' && isFollowingString !== 'false')
```

Comment:
This is very nice! Sadly it cannot verify isFollowingString as type string, so I have to do this:
```
if (
        typeof userId !== 'string' ||
        typeof isFollowingString !== 'string' ||
        !['true', 'false'].includes(isFollowingString)
    )
```
and then also isFollowingString is just a string and not type 'true' | 'false' as it is after this check 
`isFollowingString !== 'true' && isFollowingString !== 'false'`

Is both possible, just the latter one shorter in the end since no typeof check is needed